classes/opponentSpawner.lua

```diff
@@ -9,13 +9,10 @@ function opponentSpawner.new()
 	self.index = 1
 	self.list =
 	{
-		classes.opponentPhaseNormal,
-		classes.opponentPhaseNormal,
 		classes.opponentPhaseNormal,
 		classes.opponentPhaseBouncing,
-		classes.opponentPhaseBouncing,
-		classes.opponentPhaseNoBehavior,
```

Comment:
Either:

1. Revert these lines of code so that each phase has multiple copies like they used to, OR

2. Delete the extra `classes.opponentPhaseSpin` so that each class appears exactly once, so testing goes by faster. 

Honestly, I prefer the second one.